app/controllers/UserController.scala

```diff
@@ -1,24 +1,48 @@
 package controllers
 
-import scala.concurrent.Future
+import scala.concurrent.{ ExecutionContext, Future }
 
 import com.mohiva.play.silhouette.api.actions.SecuredRequest
 import com.mohiva.play.silhouette.api.Silhouette
 import javax.inject._
+import play.api.i18n.I18nSupport
 import play.api.libs.json._
 import play.api.mvc._
 
-import models.User
+import forms.UpdateUserForm
+import models.{ User, UserType }
+import models.services.UserService
 import utils.auth.DefaultEnv
 
 class UserController @Inject() (
   cc: ControllerComponents,
-  silhouette: Silhouette[DefaultEnv]
-) extends AbstractController(cc) {
+  silhouette: Silhouette[DefaultEnv],
+  userService: UserService
+)(
+  implicit
```

Comment:
haven't we been generally keeping these on one line? e.g. `(implicit ec: ExecutionContext)`, not really a big deal